src/main/java/uk/gov/companieshouse/api/accounts/model/ixbrl/balancesheet/FixedAssets.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.companieshouse.api.accounts.model.ixbrl.balancesheet;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class FixedAssets {
+
+    @JsonProperty("tangible")
+    public TangibleAssets tangibleAssets;
+
+    @JsonProperty("current_total")
+    public int totalFixedAssetsCurrent;
+
+    @JsonProperty("previous_total")
+    public int totalFixedAssetsPrevious;
+
+    public TangibleAssets getTangibleAssets() { return tangibleAssets; }
```

Comment:
This can be put over multiple lines to be consistent with the other boilerplate methods in the file.